criu/mount.c

```diff
@@ -95,12 +95,12 @@ int ext_mount_parse_auto(char *key)
 static char *ext_mount_lookup(char *key)
 {
 	char *v;
-	int len = strlen(key);
+int len = strlen(key);
```

Comment:
> clang-format: Possible codding-style problem

can it give more info about a problem?

Reply:
I feel that hiding clang-generated auto-formats is a good thing, else we would end up blindly applying them again and code readability would suffer 😀 It should be enough to point the line in file to make manual review a bit easier.

Moreover, multiline warnings seem to be not working on github https://github.com/actions/toolkit/issues/193#issuecomment-1691046058 . And also there is no way to get info from clang about type of error.